src/main/java/com/google/sps/data/Election.java

```diff
@@ -132,8 +133,9 @@ public Election fromVoterInfoQuery(DatastoreService datastore, JSONObject voterI
       throws JSONException {
     Set<Long> contestKeyList = this.getContests();
     Set<Long> referendumKeyList = this.getReferendums();
-    ArrayList<EmbeddedEntity> pollingStations =
-        new ArrayList<EmbeddedEntity>(this.getPollingStations());
+
+    List<EmbeddedEntity> pollingStations = new ArrayList<EmbeddedEntity>(this.getPollingStations());
```

Comment:
You can remove the type from the instantiation now, java got smarter.

new ArrayList<>(this.getPollingStations());

Reply:
Got it, will do!